app/services/movie_data_service.rb

```diff
@@ -250,5 +249,35 @@ def get_common_movies_between_multiple_actors(actor_names: nil, paginate_actor_n
         total_pages: movie_response[:total_pages]
       )
     end
+
+    def get_movie_streaming_service_providers(movie)
+      results = Tmdb::Client.request(:streaming_service_providers, movie_id: movie.tmdb_id).dig(:results, :US)
+      preferred_providers = [
```

Comment:
I got to move the name and url data about each provider out of the view (hardcoded) 😄 and into something more useful here in this object. Even if we weren't doing this fancy API stuff, I'd be tempted to at least do this.